support-frontend/app/controllers/Promotions.scala

```diff
@@ -30,7 +30,7 @@ class Promotions(
 
   implicit val a: AssetsResolver = assets
 
-  def promo(promoCode: String): Action[AnyContent] = CachedAction() { implicit request =>
+  def promo(promoCode: String, orderIsAGift: Boolean = false): Action[AnyContent] = CachedAction() { implicit request =>
```

Comment:
It doesn't look like this parameter is actually used anywhere, could it be removed?

Reply:
Absolutely! Thanks Rupert 👍